### ISSUE TITLE
up-to-date action should be activated by auto-merge-bot

### ISSUE THREAD

**Issue by rzadp** [paritytech/up-to-date-action#14]
The up-to-date action is activated by a push to master, and updates PRs that activated the auto-merge-bot.

However if a PR activates the auto-merge-bot and is not updated with master at the time, it won't be updated until another PR gets merged.

We should try to update it right away.
```[tasklist]
### Tasks
- [ ] https://github.com/paritytech/auto-merge-bot/issues/23
```

**Comment by Bullrich** [comment]
Should be doable by adding `auto_merge_enabled` [event](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target).

**Comment by Bullrich** [comment]
Or even better, by subscribing the action into the event:
```yml
  workflow_run:
    workflows:
      - auto-merge-bot
    types:
      - completed
```

This way it automatically triggers when `auto-merge-bot` was triggered.

**Comment by Bullrich** [comment]
Currently blocked by paritytech/up-to-date-action#16 as currently it will try to merge already up-to-date PRs and throw error in all the pre-existing PRs
